### ISSUE TITLE
Greif Client Pro

### ISSUE THREAD

**Issue by Koriik** [Hellsing/ModAPI-Mods#11]
Hey, Hellsing I see you havn't updated this mod In awhile. Was wondering if you could fix a few features for it, I check your steam account saw it was vac banned :( I'm sorry. you prolly don't play steam anymore or have no interest in this mod anymore, but it would be amazing if you could fix a few issues with it. -You can get banned and kicked, your name is in the menu. the sphere doesn't work anymore :(. The kill feature doesn't work. Just a few things about it. but the coolest one would be just not getting banned anymore >:D. Im not always evil but sometimes I feel like having fun :'D

**Comment by Hellsing** [comment]
Hey there.

The VAC ban is not due to The Forest, it happened in Rust back when the game was shit and there was no anti-cheat. When I last modded The Forest there was no VAC protection and I doubt that there is any right now. They claimed they added VAC protection to servers back then but even that was not preventing me from joining with modified game files. Last time I checked the anti-ban feature I saw that they added a method that can't be avoided anymore so that can't be restored, ban/kick means 100% disconnection to the host now. Also I'm personally not motivated to continue developing this project, which is why it's outdated for so long now. So either you fork the project and give it a try yourself or you gotta find someone else to update the remaining features.

Best of luck.
